app/forms/provider_interface/provider_relationship_permissions_setup_wizard.rb

```diff
@@ -89,12 +86,10 @@ def next_step
     def previous_step
       if checking_answers.present?
         [:check]
-      elsif current_step == 'info'
-        [:organisations]
       elsif current_step == 'organisations'
         [:start]
```

Comment:
A bit confused about `:start` here, what does this map to?

Reply:
:+1: Yes I think this is misleading as it implies there is a corresponding step. This case could be removed as the return value is never used  as there is no back link on the orgs step. Will simplify in this PR.